server/kolide/osquery.go

```diff
@@ -78,27 +78,3 @@ type PermissivePackContent struct {
 type Packs map[string]PackContent
 
 type PermissivePacks map[string]PermissivePackContent
-
-// Decorators is the format of the decorator configuration in an osquery config.
-type Decorators struct {
-	Load     []string            `json:"load,omitempty"`
-	Always   []string            `json:"always,omitempty"`
-	Interval map[string][]string `json:"interval,omitempty"`
-}
-
-// OsqueryConfig is a struct that can be serialized into a valid osquery config
-// using Go's JSON tooling.
-type OsqueryConfig struct {
```

Comment:
Feels weird to lose this. Though it's probably better in the go sdk anyhow.

Reply:
It's actually been unused since Fleet 2.0. It's up to the user to put in a valid config (via yaml) and we don't have to do anything to it besides serialize to JSON.